### ISSUE TITLE
[WebSocket] syndicateMissions jobs rewardPool not populated

### ISSUE THREAD

**Issue by KingPrimes** [WFCD/warframe-status#1320]
# websocket example：

``` json
{
 "syndicateMissions":[
   {
                    "id": "1706269353345SolarisSyndicate",
                    "activation": "2024-01-26T09:12:34.471Z",
                    "startString": "-1h 45m 20s",
                    "expiry": "2024-01-26T11:42:33.345Z",
                    "active": true,
                    "syndicate": "Solaris United",
                    "syndicateKey": "Solaris United",
                    "nodes": [],
                    "jobs": [
                        {
                            "id": "VenusPreservationJobDefense1706269353345",
                            "rewardPool": [],
                            "type": "Picket Duty",
                            "enemyLevels": [
                                5,
                                15
                            ],
                            "standingStages": [
                                350,
                                350,
                                350
                            ],
                            "minMR": 0,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "VenusChaosJobAssassinate1706269353345",
                            "rewardPool": [],
                            "type": "Scorched Earth",
                            "enemyLevels": [
                                10,
                                30
                            ],
                            "standingStages": [
                                720,
                                720,
                                720
                            ],
                            "minMR": 1,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "VenusHelpingJobSpy1706269353345",
                            "rewardPool": [],
                            "type": "Dog Boards",
                            "enemyLevels": [
                                20,
                                40
                            ],
                            "standingStages": [
                                610,
                                610,
                                610,
                                890
                            ],
                            "minMR": 2,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "VenusIntelJobResource1706269353345",
                            "rewardPool": [],
                            "type": "Operational Intelligence",
                            "enemyLevels": [
                                30,
                                50
                            ],
                            "standingStages": [
                                600,
                                600,
                                600,
                                600,
                                1190
                            ],
                            "minMR": 3,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "VenusCullJobExterminate1706269353345",
                            "rewardPool": [],
                            "type": "Hunter-Killer",
                            "enemyLevels": [
                                40,
                                60
                            ],
                            "standingStages": [
                                740,
                                740,
                                740,
                                740,
                                1450
                            ],
                            "minMR": 5,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "VenusPreservationJobResource1706269353345",
                            "rewardPool": [],
                            "type": "Trash Their Traps",
                            "enemyLevels": [
                                100,
                                100
                            ],
                            "standingStages": [
                                840,
                                840,
                                840,
                                840,
                                1660
                            ],
                            "minMR": 10,
                            "expiry": "2024-01-26T11:42:33.345Z"
                        },
                        {
                            "id": "NarmerVenusTheftJobExcavation1706269353345",
                            "rewardPool": [],
                            "type": "Digging for Narmer (Narmer)",
                            "enemyLevels": [
                                50,
                                70
                            ],
                            "standingStages": [
                                820,
                                820,
                                820,
                                820,
                                1610
                            ],
                            "minMR": 0,
                            "expiry": "2024-01-26T11:42:33.345Z",
                            "timeBoound": "night"
                        }
                    ],
                    "eta": "44m 38s"
                }
  ]
}
```
# https get example：

``` json
{
            "id": "1706269353345SolarisSyndicate",
            "activation": "2024-01-26T09:12:34.471Z",
            "startString": "-1h 45m 20s",
            "expiry": "2024-01-26T11:42:33.345Z",
            "active": true,
            "syndicate": "Solaris United",
            "syndicateKey": "Solaris United",
            "nodes": [],
            "jobs": [
                {
                    "id": "VenusPreservationJobDefense1706269353345",
                    "rewardPool": [
                        "100X Cryotic",
                        "1,500 Credits Cache",
                        "50 Endo",
                        "5X Thermal Sludge",
                        "2X Training Debt-Bond",
                        "Garuda Chassis Blueprint",
                        "5X Tepa Nodule",
                        "3,000 Credits Cache",
                        "Aya"
                    ],
                    "type": "Picket Duty",
                    "enemyLevels": [
                        5,
                        15
                    ],
                    "standingStages": [
                        350,
                        350,
                        350
                    ],
                    "minMR": 0,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "VenusChaosJobAssassinate1706269353345",
                    "rewardPool": [
                        "15X Thermal Sludge",
                        "200X Cryotic",
                        "2,500 Credits Cache",
                        "100 Endo",
                        "2X Shelter Debt-Bond",
                        "Garuda Systems Blueprint",
                        "5X Tepa Nodule",
                        "Aya",
                        "Mecha Overdrive"
                    ],
                    "type": "Scorched Earth",
                    "enemyLevels": [
                        10,
                        30
                    ],
                    "standingStages": [
                        720,
                        720,
                        720
                    ],
                    "minMR": 1,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "VenusHelpingJobSpy1706269353345",
                    "rewardPool": [
                        "Unairu Lens",
                        "200 Endo",
                        "300X Plastids",
                        "2X Medical Debt-Bond",
                        "Garuda Neuroptics Blueprint",
                        "3,000 Credits Cache",
                        "Aya",
                        "Mecha Recharge"
                    ],
                    "type": "Dog Boards",
                    "enemyLevels": [
                        20,
                        40
                    ],
                    "standingStages": [
                        610,
                        610,
                        610,
                        890
                    ],
                    "minMR": 2,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "VenusIntelJobResource1706269353345",
                    "rewardPool": [
                        "Zenurik Lens",
                        "300 Endo",
                        "2X Detonite Injector",
                        "2X Advances Debt-Bond",
                        "Aya",
                        "Tellurium",
                        "Mecha Empowered"
                    ],
                    "type": "Operational Intelligence",
                    "enemyLevels": [
                        30,
                        50
                    ],
                    "standingStages": [
                        600,
                        600,
                        600,
                        600,
                        1190
                    ],
                    "minMR": 3,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "VenusCullJobExterminate1706269353345",
                    "rewardPool": [
                        "400 Endo",
                        "2X Familial Debt-Bond",
                        "10,000 Credits Cache",
                        "Aya",
                        "500X Kuva",
                        "Mecha Pulse"
                    ],
                    "type": "Hunter-Killer",
                    "enemyLevels": [
                        40,
                        60
                    ],
                    "standingStages": [
                        740,
                        740,
                        740,
                        740,
                        1450
                    ],
                    "minMR": 5,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "VenusPreservationJobResource1706269353345",
                    "rewardPool": [
                        "400 Endo",
                        "2X Familial Debt-Bond",
                        "10,000 Credits Cache",
                        "Aya",
                        "500X Kuva",
                        "Mecha Pulse"
                    ],
                    "type": "Trash Their Traps",
                    "enemyLevels": [
                        100,
                        100
                    ],
                    "standingStages": [
                        840,
                        840,
                        840,
                        840,
                        1660
                    ],
                    "minMR": 10,
                    "expiry": "2024-01-26T11:42:33.345Z"
                },
                {
                    "id": "NarmerVenusTheftJobExcavation1706269353345",
                    "rewardPool": [
                        "Boreal's Anguish",
                        "Narmer Isoplast",
                        "600 Endo",
                        "Caliban Chassis Blueprint",
                        "Nira's Hatred",
                        "2X Narmer Isoplast",
                        "900 Endo",
                        "Amar's Contempt",
                        "Nepheri Blueprint",
                        "3X Narmer Isoplast",
                        "1200 Endo"
                    ],
                    "type": "Digging for Narmer (Narmer)",
                    "enemyLevels": [
                        50,
                        70
                    ],
                    "standingStages": [
                        820,
                        820,
                        820,
                        820,
                        1610
                    ],
                    "minMR": 0,
                    "expiry": "2024-01-26T11:42:33.345Z",
                    "timeBoound": "night"
                }
            ],
            "eta": "44m 37s"
        }
```

**Comment by TobiTenno** [comment]
(copying from discord)
> this is probably an issue of the reward name being an async thing to populate and `:get` does the pull immediately without a chance to make sure the reward pool is updated

**Comment by TobiTenno** [comment]
once i fix #1301, i'll probably have time to work on doing the whole post processing for the worldstate so the field is more stable for both ws & http

**Comment by TobiTenno** [comment]
@KingPrimes it's fixed, rewardPool now populates on the websocket
